app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestRoomsController < ApplicationController
+  def create
+    guest = GuestRoom.create!(guest_id: params[:id], room_id: params[:room_id])
```

Comment:
doesn't look like you're using this `guest` variable in this method, so feel free to get rid of it!